lib/redux/utils/errors.js

```diff
@@ -67,6 +67,8 @@ export function getLoginErrorMessage(status) {
 
 export function getUpdateProfileErrorMessage(status) {
   switch (status) {
+    case 400:
+      return 'Something looks funky, make sure this account info is correct.';
```

Comment:
It's the same error message as https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-ed50da20752781857ba122369a967365R84 which is presenting the same error, so I think we're good.